cosmetics-web/app/validators/accept_and_submit_validator.rb

```diff
@@ -1,9 +1,43 @@
 class AcceptAndSubmitValidator < ActiveModel::Validator
-  def validate(record)
-    missing_nano_materials = record.missing_nano_materials
+  def validate(notification)
+    validate_nano_materials(notification)
+    validate_image_uploads(notification)
+    validate_frame_formulation_uploads(notification)
+  end
+
+  def validate_nano_materials(notification)
+    missing_nano_materials = notification.missing_nano_materials
     if missing_nano_materials.present?
       missing_nano_materials.each do |nano|
-        record.errors.add :base, "#{nano.name} is not included in any items"
+        notification.errors.add :nano_materials, "#{nano.name} is not included in any items"
+      end
+    end
+  end
+
+  def validate_image_uploads(notification)
+    notification.image_uploads.each do |image_upload|
+      if image_upload.pending_antivirus_check?
+        notification.errors.add :image_uploads, "Image #{image_upload.file.filename} is still being processed"
+      end
+      if image_upload.failed_antivirus_check?
```

Comment:
:raising_hand_man: These two conditions are mutually exclusive:

```suggestion
      elsif image_upload.failed_antivirus_check?
```